analyse_DEcal_particles.py

```diff
@@ -0,0 +1,267 @@
+from ROOT import TH1F, TTree, TChain, TFile, TF1, TH2F, TGraphErrors, TCanvas, TPaveText
```

Comment:
That file should not be in the main directory. Please clean it up, comment as much as possible, remove the paths to your code (any possible inputs like e.g. file names can be entered as arguments [see here](https://github.com/vvolkl/FCCSW/blob/e42ee055e7fddebee9f5e52c7bb19212ea5b3cda/Reconstruction/RecTracker/options/TrickTrackReco.py#L7)). This could probably go to `Detector/DetectorFCChhECalDigital/scripts/`.